features/0360-use-did-key/README.md

```diff
@@ -0,0 +1,106 @@
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-12-17
+- Status Note: Socialized with mixed support; did:key method is a [W3C CCG work item](https://w3c-ccg.github.io/community/work_items.html)
+- Supersedes:
+- Start Date: 2019-12-17
+- Tags: feature
+
+## Summary
+
+A number of RFCs that have been defined reference what amounts to a "naked" public key, such that the sender relies on the reciever knowing what type the key is and how it can be used. The application of this RFC will result in the replacement of "naked" verkeys (public keys) in some DIDComm/Aries protocols with the [did:key](https://digitalbazaar.github.io/did-method-key/) ledgerless DID method, a format that concisely conveys useful information about the use of the key, including the public key type. While `did:key` is less a DID method than a transformation from a public key and type to an opinionated DIDDoc, it provides a versioning mechanism for supporting new/different cryptographic formats and its use makes clear how a public key is intended to be used. The method also enables support for using standard [DID resolution](https://w3c-ccg.github.io/did-resolution/) mechanisms that may simplify the use of the key.
+
+While it is well known in the Aries community that `did:key` is fundamentally different from the [did:peer](https://dhh1128.github.io/peer-did-method-spec/index.html) method that is the basis of Aries protocols, it must be re-emphasized here. This RFC does **NOT** imply any changes to the use of `did:peer` in Aries, nor does it change the content of a `did:peer` DIDDoc. This RFC only changes references to plain public keys in the JSON of some RFCs to use `did:key` in place of a plain text string.
+
+Should this RFC be [ACCEPTED](/README.md#accepted), a [community coordinated update](../../concepts/0345-community-coordinated-update/README.md) will be used to apply updates to the agent code bases and impacted RFCs.
+
+## Motivation
+
+When one Aries agent inserts a public key into the JSON of an Aries message (for example, the `~service` decorator), it assumes that the recipient agent will use the key in the intended way. At the time this RFC is being written, this is easy because only one key type is in use by all agents. However, in order to enable the use of different cryptography algorithms, the public references must be extended to at least include the key type. The preferred and concise way to do that is the use of the [multicodec](https://github.com/multiformats/multicodec) mechanism, which provides a registry of encodings for known key types that are prefixed to the public key in a standard and concise way. [did:key](https://digitalbazaar.github.io/did-method-key) extends that mechanism by providing a templated way to transform the combination of public key and key type into a [DID-standard DIDDoc](https://digitalbazaar.github.io/did-method-key/#example-2-a-did-document-derived-from-a-did-key).
```

Comment:
hopefully the W3C repo becomes populated: 
- https://github.com/w3c-ccg/did-method-key
- https://w3c-ccg.github.io/did-method-key

(should have a TODO to update the references & links, once available.)

Reply:
Added To Do.